phoenix-core/src/it/java/org/apache/phoenix/execute/PartialCommitIT.java

```diff
@@ -259,10 +259,10 @@ private PhoenixConnection getConnectionWithTableOrderPreservingMutationState() t
         Connection con = driver.connect(url, new Properties());
         PhoenixConnection phxCon = new PhoenixConnection(con.unwrap(PhoenixConnection.class));
         final Map<TableRef,Map<ImmutableBytesPtr,MutationState.RowMutationState>> mutations = Maps.newTreeMap(new TableRefComparator());
-        return new PhoenixConnection(phxCon) {
+        return new PhoenixConnection(phxCon, null) {
```

Comment:
Never mind on this change - passing null is actually required here, though might be worth a code comment on why (because it forces the connection.newMutationState() to be used to create the MutationState).